sdcm/nemesis.py

```diff
@@ -2680,12 +2680,12 @@ def add_new_node(self, rack=0):
     def decommission_node(self, node):
         self.cluster.decommission(node)
 
-    def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[bool, DefaultValue]] = DefaultValue):
+    def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[bool, DefaultValue]] = DefaultValue, dc_idx: Optional[Union[int, None]] = None):
```

Comment:
AFAIU `Optional[Union[int, None]]` is redundant, as it is the same as `Union[int, None]` with the same effect.

Reply:
Thanks. fixed